etc/kayobe/kolla/config/ironic/ironic-conductor.conf

```diff
@@ -0,0 +1,92 @@
+[DEFAULT]
+# Make direct deploy faster, transfer sparse qcow2 images
+force_raw_images = False
+# Avoid some rpc timeouts
+rpc_response_timeout = 360
+
+[agent]
+# Use iPXE server for direct deploy source
+image_download_source = http
+# Reduce load by only collecting logs on failure
+deploy_logs_collect = on_failure
+
+[console]
+socat_address = {% raw %}{{ api_interface_address }}{% endraw %}
+
+[conductor]
+automated_clean=true
+# We have busy conductors failing to heartbeat
+# Default is 10 secs
+heartbeat_interval = 30
+# Default is 60 seconds
+heartbeat_timeout = 360
+sync_local_state_interval = 360
+
+[agent]
+# Use iPXE server for direct deploy source
+image_download_source = http
+# Reduce load by only collecting logs on failure
+deploy_logs_collect = on_failure
+
+[console]
+socat_address = {% raw %}{{ api_interface_address }}{% endraw %}
+
+[conductor]
+automated_clean=true
+# We have busy conductors failing to heartbeat
+# Default is 10 secs
+heartbeat_interval = 30
+# Default is 60 seconds
+heartbeat_timeout = 360
+sync_local_state_interval = 360
+
+# Normally this is 100. We see eventlet threads
+# not making much progress, to for saftey reduce
+# this by half, should leave work on rabbit queu
+workers_pool_size = 50
+# Normally this is 8, keep it same
+period_max_workers = 8
+
+# Stop power state sync
+sync_power_state_interval = 0
+power_failure_recovery_interval = 0
+# Stop checking for orphan allocations for now
+check_allocations_interval = 0
+
+# Wait much longer before provision timeout check, to reduce background load
+# The default is 60 seconds
+check_provision_state_interval = 120
+check_rescue_state_interval = 120
+
+[database]
```

Comment:
I think we have some of this in upstream kolla now, from the work Pierre did with failovers?